Makefile

```
@@ -228,3 +228,45 @@ disable-maintenance: read-tf-config ## make dev disable-maintenance / make produ
 	echo Waiting 5s for route to be registered... && sleep 5
 	cf unmap-route apply-for-qts-unavailable london.cloudapps.digital --hostname apply-for-qts-in-england-${DEPLOY_ENV}
 	cf delete apply-for-qts-unavailable -r -f
+
+validate-domain-resources: set-what-if domain-azure-resources # make publish validate-domain-resources AUTO_APPROVE=1
+
+deploy-domain-resources: check-auto-approve domain-azure-resources # make publish deploy-domain-resources AUTO_APPROVE=1
+
+.PHONY: afqts_domain
+afqts_domain:   ## runs a script to config variables for setting up dns
+	$(eval include global_config/afqts_domain.sh)
```

```suggestion
.PHONY: domain
domain:   ## runs a script to config variables for setting up dns
	$(eval include global_config/domain.sh)
```

```suggestion
.PHONY: domain
domain:   ## runs a script to config variables for setting up dns
```

There's no need to include a `afqts_` prefix in this Makefile since we're already within the Apply for QTS project. It's worth noting that we only call it `afqts` where Azure limits the length of our names, and it's not commonly used as an acronym on this project otherwise.

---

script name shorten as suggested